clean_enoa/call_inegi.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import os
+import json
+
+
+def output(content, index):
+    per = content["Series"][0]["OBSERVATIONS"][index]["TIME_PERIOD"]
+    val = content["Series"][0]["OBSERVATIONS"][index]["OBS_VALUE"]
+    print(per, val)
+
+
+def get_content(url):
+    response = requests.get(url)
+    if response.status_code == 200:
+        content = json.loads(response.content)
+    return content
+
+
+def desc_grupo(content, grupo):
+    ocupado_2021 = int(content["Series"][0]["OBSERVATIONS"][0]["OBS_VALUE"])
+    ocupado_2020 = int(content["Series"][0]["OBSERVATIONS"][4]["OBS_VALUE"])
+    salida = f"La población {grupo} fue de {ocupado_2021}, cifra menor en {ocupado_2021 - ocupado_2020} con respecto al mismo trimestre del año anterior."
+    print(salida)
+
+
+inegi_token = os.environ["INEGI_TOKEN"]
+
+
+def get_trimester_pea(trimester):
+    pea = f"https://www.inegi.org.mx/app/api/indicadores/desarrolladores/jsonxml/INDICATOR/289244/es/0700/false/BIE/2.0/{inegi_token}?type=json"
```

Comment:
¿Qué significa la abreviatura PEA? ¿Población Económicamente Activa?
```suggestion
    poblacion_economicamente_activa = f"https://www.inegi.org.mx/app/api/indicadores/desarrolladores/jsonxml/INDICATOR/289244/es/0700/false/BIE/2.0/{inegi_token}?type=json"
```

Reply:
Sí, eso significa y ya lo cambié :raised_hands: